### ISSUE TITLE
Import to Calibre Failing

### ISSUE THREAD

**Issue by Viper911** [DobyTang/LazyLibrarian#1189]
Thoughts on waht could be the issue.  I am using the "server" type connection to calibre and it passes the user and pw just find, but cant write and log show forbidden in the post procssing.

In setting I run the test and get
READ Pssed
WRITE failed

In the logon post processing I get

2018-01-13 08:27:34 | WARNING | POSTPROCESS | postprocess.py | import_book | 966 | Residual files remain in /CompHD/completed/Books/Jacob Grimm - Grimms Fairy Tales (Annotated) LL.(25244403).fail
-- | -- | -- | -- | -- | -- | --
2018-01-13 08:27:34 | ERROR | POSTPROCESS | postprocess.py | import_book | 958 | Postprocessing for Grimms' Fairy Tales (Annotated) - Jacob Grimm has failed: calibredb rc 1 from /opt/calibre/calibredb
2018-01-13 08:27:34 | DEBUG | POSTPROCESS | postprocess.py | processDestination | 1112 | /opt/calibre/calibredb result : Another calibre program such as calibre-server or the main calibre program is running. Having multiple programs that can make changes to a calibre library running at the same time is a bad idea. calibredb can connect directly to a running calibre content server, to make changes through it, instead. See the documentation of the --with-library option for details.
2018-01-13 08:27:34 | DEBUG | POSTPROCESS | calibre.py | calibredb | 452 | calibredb retry returned 1: res[Another calibre program such as calibre-server or the main calibre program is running. Having multiple programs that can make changes to a calibre library running at the same time is a bad idea. calibredb can connect directly to a running calibre content server, to make changes through it, instead. See the documentation of the --with-library option for details. ] err[]
2018-01-13 08:27:33 | DEBUG | POSTPROCESS | calibre.py | calibredb | 446 | [u'/opt/calibre/calibredb', 'add', '-1', '--with-library', u'/media/SharedFiles/Books/', u'/CompHD/completed/Books/Jacob Grimm - Grimms Fairy Tales (Annotated) LL.(25244403)']
**2018-01-13 08:27:33 | DEBUG | POSTPROCESS | calibre.py | calibredb | 431 | calibredb returned 1: res[Forbidden ] err[]**
2018-01-13 08:27:33 | DEBUG | POSTPROCESS | calibre.py | calibredb | 420 | [u'/opt/calibre/calibredb', 'add', '--username', u'htpc', '--password', u'htpc', '-1', '--with-library', u'http://192.168.1.33:8080', u'/CompHD/completed/Books/Jacob Grimm - Grimms Fairy Tales (Annotated) LL.(25244403)']
2018-01-13 08:27:33 | DEBUG | POSTPROCESS | postprocess.py | processDestination | 1086 | Importing Grimms' Fairy Tales (Annotated) - Jacob Grimm into calibre library
2018-01-13 08:27:33 | DEBUG | POSTPROCESS | postprocess.py | import_book | 878 | Bookid 25244403 was not snatched so cannot check type, contains eBook
2018-01-13 08:27:33 | DEBUG | POSTPROCESS | postprocess.py | processDir | 713 | Book with id: 25244403 found in download directory

**Comment by philborman** [comment]
You seem to be talking to the calibre server ok, but probably have a calibre auth issue. Without auth you can read the database, but writing to it generates "Forbidden".  

Things to check...
calibre server needs starting with the --enable-auth option, or if using the main calibre gui you need to go to preferences->sharing over the net and tick "require username and password to access the server"

you also need to create a user/password in calibre preferences->sharing over the net->user accounts and give the same user/password to lazylibrarian so it can talk to calibre

**Comment by philborman** [comment]
closing, if still not working please re-open with more info

**Comment by ghost** [comment]
Was this ever fixed? I have Calibre in docker on one machine, with LazyLibrarian on another machine. LazyLibrarian post processing fails, history tab shows failed items with error "Error: calibredb rc 1 from ". In LazyLibrarian settings, calibredb is blank, because it was suggested before to no longer use that method, to use the Calibre content server instead, which I have set correctly in LazyLibrarian (same settings as Calibre-Web, which works fine). All relevant directories have 777 access.

**Comment by hannashimona** [comment]
@dalepmay7 did you ever solve? I have the exact same issue.

**Comment by MrMxyzptlk** [comment]
Yes, it was all me. I thought I had updated to the latest LL, but I hadn't. The bigger issue was I had reinstalled Calibre and forgot to readd the user for the webserver, so LL wasn't unable to communicate with Calibre. Once I fixed that, all is well.

Here are my settings...you will need to start the Calibre Content Server in calibre as well as add a user to the CCS in Calibre settings.

![Screenshot 2025-01-09 at 9 15 17 AM](https://github.com/user-attachments/assets/f6e726fb-6abc-4bc0-8314-17a13dbf6d89)
![Screenshot 2025-01-09 at 9 15 29 AM](https://github.com/user-attachments/assets/430ab7b1-2de9-4db3-a263-cbc07596a95b)
